storage/storage.go

```diff
@@ -532,3 +534,8 @@ func CreateSendLog(c context.Context, sendLogs *entities.SendLog) error {
 func GetSendLogByUUID(c context.Context, id string) (*entities.SendLog, error) {
 	return GetFromContext(c).GetSendLogByUUID(id)
 }
+
+// CreateSubscribersEvent adds new subscribers event record in database
+func CreateSubscribersEvent(c context.Context, se *entities.SubscribersEvent) error {
```

Comment:
The writing of subscribers event should always be accompanied by the actual event in a transaction, I'm thinking that we won't need this method right?

Reply:
Yes you are right this should be added than on create subscriber delete or unsubscribe. Then i will delete this store method.